lib/dbManager.php

```diff
@@ -27,22 +27,28 @@ function __construct ($dbFilename, $flags = SQLITE3_OPEN_READWRITE) {
     // @param isAdmin: optionnal, specifies if the user is an admin. Defaults to 0. 
     // @return bool : TRUE if no error, FALSE otherwise
     public function addUser ($username, $password, $gauthSecret, $isAdmin = 0) {
+
+		// Prepare variables before the query
+		$passwordHash = hash('sha256',$password);
         
-        // Prepare variables before the query
-        $username = SQLite3::escapeString ($username);
```

Comment:
As I mentioned, there is a bug in the SQLite3 library: https://bugs.php.net/bug.php?id=62361
Also see: http://security.stackexchange.com/questions/3611/sql-injection-why-isnt-escape-quotes-safe-anymore

So considering both factors, I replaced all the escapeString calls with prepared statements, which don't need to be escaped and are more secure.